libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -260,6 +260,11 @@ export class SkyDateRangePickerComponent
   protected showEndDatePicker = signal<boolean>(false);
   protected showStartDatePicker = signal<boolean>(false);
 
+  #_calculatorIds = SKY_DEFAULT_CALCULATOR_IDS;
+  #_label: string | undefined;
+  #_value = this.selectedCalculator.getValue();
```

Comment:
Here's the primary change; it needs to get the default value before creating the form controls.